languages/python/exercises/concept/lists/lists.py

```diff
@@ -0,0 +1,46 @@
+def to_list(arg1, arg2, arg3, arg4, arg5):
```

Comment:
I might push the conceit of the  exercise narrative here a bit and use card_1 .... card_5 for the argument names.